src/test/java/com/scriptbasic/testprograms/TestPrograms.java

```diff
@@ -169,25 +171,25 @@ public void tooLargeArrayCannotBeCreated() {
 
     @Test()
     public void syntaxErrorWhenThereIsNoClosingParentheseAfterFunctionCall() {
-        Assertions.assertThrows(BasicRuntimeException.class, () ->
+        Assertions.assertThrows(CommandFactoryException.class, () ->
```

Comment:
I do not understand how this is possible that this test passes and at the same time the `master` branch also passes the tests. These are two different exception types. Please help me.

Eventually, it seems that the exception should NOT be `BasicRuntimeException` because it is a syntax analysis error test, and still it does not break the `master` build. Strange. Can you help me to figure it out?